docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -127,33 +117,31 @@ You need the following to get invited to TechPass and complete the onboarding:
 
   ![vendor-mfa-9](assets/support/vendor-mfa-9-new.png)
 
-  Now you will be prompted to reset your initial password.
-</details>
+  Now you are now prompted to reset your initial password.
 
-## Step 4. Reset initial password
 
-<details> <summary style="font-size:20px;font-weight:bold"> Reset the initial password</summary>
+## Step 4. Reset initial password
 
-  1. Enter your **initial password**, **new password** and retype the new password to confirm.
+ 1. Enter your **initial password**, **new password** and retype the new password to confirm.
 
   2. Click **Sign in** to proceed to accept the Terms of Use.
 
   ![vendor-mfa-9](assets/support/vendor-update-initial-password.png)
-</details>
+
 
 ## Step 5. Accept Privacy Policy and Terms of Use
```

Comment:
I thought it should read, but i may be wrong
"Accept the Privacy Policy and Terms of Use"

Reply:
I can club them together as suggested but these 2 policies are listed one after the other, so listed them separately. If you your prefer to showm them in one step, i will do it.